### ISSUE TITLE
Searching for vim-x11 returns 0 results

### ISSUE THREAD

**Issue by gsauthof** [fedora-infra/fedora-packages#229]
When searching for 'vim-x11' or 'vim-X11' 0 results are returned:

https://apps.fedoraproject.org/packages/s/vim-X11

But `dnf search vim-X11` returns something on a Fedora system:

    vim-X11.x86_64 : The VIM version of the vi editor for the X Window System

**Comment by cverna** [comment]
Thanks @nirik for going through the tickets. For now code wise we should be I am just waiting on https://src.fedoraproject.org/rpms/TurboGears2/pull-request/1 to be reviewed and merged.

**Comment by nirik** [comment]
reviewed and merged that and made f26/f27/rawhide builds.

**Comment by avsej** [comment]
Will it pickup all new packages which were added while fedora-packages app was broken?

**Comment by cverna** [comment]
@avsej yes it will, The verison which is currently in production is using pkgdb to get the list of packages, and as you might know pkgdb has been retired and it is not updated. The new verison of fedora-packages is using [pdc](https://pdc.fedoraproject.org/) to get the list of packages.

**Comment by avsej** [comment]
@cverna any plans when new version will be deployed?

**Comment by cverna** [comment]
@avsej Hopefully in the next couple of weeks. As you may know there are currently some maintenance work in the fedora-infrastructure. Once these works are finished we will be able to deploy the latest release.

**Comment by junghans** [comment]
lammps (#307) has a package page now, but newer packages, e.g. mpibash are still missing.

**Comment by cverna** [comment]
yes i think the mechanism to update the packages is not working :(. I ll try to investigate why

**Comment by pypingou** [comment]
Shot in the dark, it's listening for the fedmsg messages from pkgdb?

**Comment by pypingou** [comment]
hm, that looks problematic: https://apps.fedoraproject.org/datagrepper/raw?topic=org.fedoraproject.prod.mdapi.repo.update

**Comment by cverna** [comment]
So I think we know why this is not working :), any ideas why the mdapi.repo.update fedmsg is not sent anymore ?

**Comment by pypingou** [comment]
mdapi is sending messages again, let's see if fedora-packages catches them :)
